changelogs/current.yaml

```diff
@@ -136,16 +120,21 @@ minor_behavior_changes:
 - area: tracers
   change: |
     remove unnecessary "spawnChild" annotations in OpenCensus tracer.
+- area: conn pool
+  change: |
+    changed HTTP/2 connection pooling and the :ref:`ALPN pool <envoy_v3_api_field_extensions.upstreams.http.v3.HttpProtocolOptions.auto_config>` to remember the number of streams allowed by the endpoint and cap multiplexed streams for subsequent connections based on that. With that working, defaulted the ALPN pool to assume HTTP/2 will work, as it will only incur a latency hit once until the TLS handshake is complete, and then will cache that the effective stream limit is 1.  This behavioral change can be revered by setting ``envoy.reloadable_features.allow_concurrency_for_alpn_pool`` to false.
```

Comment:
tip for future - you can use multiline and unless there are 2 separate lines rst folds them to a ws-separated single line in output